packages/web/src/components/artist/ArtistPopover.tsx

```diff
@@ -79,6 +83,22 @@ const ArtistPopover = ({
     true
   )
 
+  const supportingMap = useSelector(getSupporting)
+  const encodedCreatorUserId = encodeHashId(creator?.user_id ?? null)
+  const supportingForCreator = encodedCreatorUserId
+    ? supportingMap[encodedCreatorUserId] ?? {}
+    : {}
+  const rankedSupportingList = Object.keys(supportingForCreator)
```

Comment:
It seems like it might be more valuable to get the list as-is from the server. It's already sorted by amounts.